packages/hardhat-core/sample-projects/advanced/hardhat.config.js

```diff
@@ -0,0 +1,42 @@
+require("dotenv").config();
+
+require("@nomiclabs/hardhat-etherscan");
+require("@nomiclabs/hardhat-waffle");
+require("hardhat-gas-reporter");
+require("solidity-coverage");
+
+const { ethers } = require("ethers");
+
+// This is a sample Hardhat task. To learn how to create your own go to
+// https://hardhat.org/guides/create-task.html
+// eslint-disable-next-line no-undef
```

Comment:
An alternative to this is to add:

```
    {
      files: ["hardhat.config.js"],
      globals: {
        task: true,
      },
    },
```

to the `.eslintrc`'s overrides.

Reply:
I like that.  Thank you.  Addressed in https://github.com/nomiclabs/hardhat/pull/1632/commits/5703d24c6e3a34d700a62b84b15980ed04e2e73e